test/companies/schema/company_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule Companies.Schema.CompanyTest do
+  @moduledoc """
+    This module holds the unit tests to the company changeset
+  """
+  use Companies.DataCase
+
+  alias Companies.{
+    Schema.Company
+  }
+
+  @valid_company_params %{
```

Comment:
We're using `ex_machina` on this project so you could have used that to generate these parameters for you.

https://hexdocs.pm/ex_machina/ExMachina.Ecto.html#c:params_for/2